privacy_evaluator/models/torch/cnn.py

```diff
@@ -4,17 +4,23 @@
 
```

Comment:
Please add documentation to this file

Reply:
The last editor can have a look at [here](https://github.com/privML/privacy-evaluator/pull/142/commits/92178ee4bc2904fbb3c3d7e827deeef5e3ad73b6), and maybe you just pull the team2sprint3 branch and do the similar.